nodejs/week1/app.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+const app = express();
+
+// Import data here
+const meals = require("./data/meals");
+const reservation = require("./data/reservations");
+const reviews = require("./data/reviews");
+const reviewedMeal = meals.map((meal) => {
+  meal.reviews = [];
+  reviews.forEach((review) => {
+    if (meal.id === review.mealId) {
+      meal.reviews.push(review);
+    }
```

Comment:
Good work using forEach method.
You can also use filter method as well.
like `meal.reviews = reviews.filter((review) => review.mealId === meal.id);`